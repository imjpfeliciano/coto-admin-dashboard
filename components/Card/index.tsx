import { WithChildren } from "../../types/components";

const Card: React.FC<WithChildren> = ({ children }) => {
  return (
    <div className="bg-white rounded-md p-4 shadow-md">
      {children}
    </div>
  )
}
export default Card;
