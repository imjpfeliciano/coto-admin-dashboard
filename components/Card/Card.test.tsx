import Card from ".";
import { render } from "@testing-library/react";

describe("Card", () => {
  it("should render", () => {
    const { container } = render(<Card />);
    expect(container).toMatchSnapshot();
  });
});
