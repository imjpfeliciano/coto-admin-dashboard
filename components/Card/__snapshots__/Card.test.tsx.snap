// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render 1`] = `
<div>
  <div
    class="Card-sc-18vqxqo-1 cIOZno"
  />
</div>
`;
